corehq/toggles/__init__.py

```diff
@@ -2458,6 +2458,14 @@ def _handle_attendance_tracking_role(domain, is_enabled):
                'can+be+used+to+send+emails+from+HQ'),
 )
 
+ABDM_INTEGRATION = StaticToggle(
+    'abdm_integration',
+    'Allows the project to use ABDM Integration',
+    TAG_CUSTOM,
+    namespaces=[NAMESPACE_DOMAIN],
+    description='This feature flag allows the project to use ABDM Integration.'
```

Comment:
We should link to the documentation here.

Reply:
Resolved in 8c0fb8c4ee14bdcac94176525d50bcd2a0c6ef6e